src/daemons/MetaDaemon.cpp

```diff
@@ -95,6 +86,25 @@ int main(int argc, char *argv[]) {
     }
     auto& localhost = hostAddrRet.value();
 
+    // folly IOThreadPoolExecutor
+    auto ioPool = std::make_shared<folly::IOThreadPoolExecutor>(FLAGS_num_io_threads);
+
+    LOG(INFO) << "Starting Meta HTTP Service";
+    nebula::WebService::registerHandler("/meta", [&localhost, &ioPool] {
```

Comment:
Besides, you are registering a HTTP handler for meta, while users definitely know this is the very interface of meta service. Then, why do you register the handler as `/meta`, instead of `/` or something else?

Reply:
When user requests this HTTP interface, I hope I can make sure that they are using the specify component's interface. `/meta` maybe better than others and `/` is too common.